clmm/dataops/__init__.py

```diff
@@ -257,76 +200,51 @@ def compute_background_probability(
 
 
 def compute_galaxy_weights(
-    z_lens,
-    cosmo,
-    z_source=None,
-    use_pdz=False,
-    pzpdf=None,
-    pzbins=None,
+    sigma_c=None,
```

Comment:
Same as above - still request the user to pass sigma_c, but have an explicit flag for them to say they want DeltaSigma weights and not shear weights?